shared/test_artifacts/generator.go

```diff
@@ -34,64 +41,64 @@ var (
 // TODO (Team) this is meant to be a **temporary** replacement for the recently deprecated
 // 'genesis config' option. We need to implement a real suite soon!
 func NewGenesisState(numValidators, numServiceNodes, numApplications, numFisherman int) (genesisState modules.GenesisState, validatorPrivateKeys []string) {
-	apps, appsPrivateKeys := NewActors(MockActorType_App, numApplications)
-	vals, validatorPrivateKeys := NewActors(MockActorType_Val, numValidators)
-	serviceNodes, snPrivateKeys := NewActors(MockActorType_Node, numServiceNodes)
-	fish, fishPrivateKeys := NewActors(MockActorType_Fish, numFisherman)
+	apps, appsPrivateKeys := NewActors(types.UtilActorType_App, numApplications)
```

Comment:
I like that we deleted the mocks!

I don't think this change is exclusive to either `v1 strings or structs` implementation however

Reply:
I tried to cater to both point of views and hacked some code to see if something in between could be feasible of not.